extra-kotlin/src/main/kotlin/net/kyori/adventure/extra/kotlin/RGBLikeExtensions.kt

```diff
@@ -28,29 +28,29 @@ import net.kyori.adventure.util.RGBLike
 /**
  * The [RGBLike.red] component.
  *
- * Allows for `(r, g, b)` value decomposition.
+ * Allows for destructuring into (red, green, blue)
  *
  * @return the [RGBLike.red] component
  * @since 4.6.0
  */
-operator fun RGBLike.component1(): Int = this.red()
+public operator fun RGBLike.component1(): Int = this.red()
 
 /**
  * The [RGBLike.green] component.
  *
- * Allows for `(r, g, b)` value decomposition.
+ * Allows for destructuring into (red, green, blue)
```

Comment:
`r, g, b` is fine, no need for the full words

Reply:
Personally, I hate abbreviations, no matter what they are, but it's not my project, so I'll respect your wishes. Not saying it doesn't make sense, or that you're an idiot or anything like that, just abbreviations make me cringe lol